csm_web/frontend/static/frontend/css/style.css

```diff
@@ -207,9 +210,76 @@ header {
 	background-color: #007bff;
 	border-radius: 10px;
 	cursor: pointer;
-	left: 0;
-	right: 0;
-	bottom: 20px;
+}
+
+.affinity-btn {
+	flex-direction: row;
+	font-size: 1.25em;
+	color: white;
+	background-color: var(--csm-green);
+	border-radius: 10px;
+	cursor: pointer;
+	padding: 10px 20px;
+	margin: 0% 2%;
+}
```

Comment:
The button border in the manage label modal doesn't really fit in with the rest of Scheduler, so I'd recommend overriding the default border by setting border-width to 0